test/src/unit-allocator.cpp

```diff
@@ -100,6 +100,9 @@ struct my_allocator : std::allocator<T>
         std::allocator<T>::deallocate(p, n);
     }
 
+    // the code below warns about p in MSVC 2015 - this could be a bug
```

Comment:
"C4100 can also be issued when code calls a destructor on a otherwise unreferenced parameter of primitive type. This is a limitation of the Microsoft C++ compiler."
https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-4-c4100?view=msvc-160

Reply:
Thanks, I fixed it.